css/styles.css

```diff
@@ -0,0 +1,167 @@
+* {
+	margin: 0;
+	padding: 0;
+	line-height: 1.5em;
+}
+
+/*color-scheme: #5fc157, black, #005100, #787f63*/
+
+body {
+	background-color: black;
+}
+.main-content {
+	width: 60%;
+	margin: 2em auto 0 auto;
+	font-family: "Exo 2";
+	color: #5fc157;
+	min-width: 660px;
+}
+nav {
+	height: 40px;
+	background-color: black;
+	border-bottom: 2px solid black;
+	min-width: 450px;
+}
+nav ul > li {
+	display: inline;
+	color: #5fc157;
+	font-family: Orbitron;
+	padding: 1em;
+	font-size: .8em;
+}
+nav ul a:visited, nav ul a:link {
+	color: #5fc157;
+}
+nav ul a {
+	text-decoration: none;
+	padding: .2em .5em .2em .5em;
+	border-radius: 8px;
+}
+nav ul a:hover, nav ul a:active {
+	color: black;
+	font-weight: bolder;
+}
+nav ul a:active {
+	background-color: #787f63;
+}
+nav ul a:hover {
+	background-color: #5fc157;
+	transition-duration: .7s;
+}
+nav ul {
+	padding-top: .5em;
+	text-align: right;
+}
+.title-image {
+	background-image: url(../images/keyboard.jpeg);
+	background-size: cover;
+	width: 100%;
+	height: 800px;
+}
+h1, h3 {
+	text-align: center;
+	color: #5fc157;
+	font-weight: bolder;
+	text-shadow: 3px 3px 8px black;
+}
+h1 {
+	font-family: 'Cormorant SC', serif;
+	font-size: 4em;
+}
+h3 {
+	font-family: Orbitron;
+	font-size: 1.3em;
+	letter-spacing: 2px;
+	padding-bottom: 1em;
+}
+.header-wrap {
+	padding-top: 200px;
+	min-width: 610px;
+}
+#content {
+	overflow: auto;
+}
+.main-content {
+	max-width: 860px;
+}
+.main-img {
+	background-image: url("../images/sadbear.jpg");
+	height: 300px;
+	min-height: 200px;
+	width: 40%;
+	background-position: center;
+	min-width: 300px;
+	background-size: cover;
+	float: left;
+	margin-right: 1em;
+	border-radius: 8px;
+}
+.title-image {
+	background-image: url(../images/keyboard.jpeg);
+	background-size: cover;
+	width: 100%;
+	height: 800px;
+}
+
```

Comment:
Try to be consistent with your line breaks throughout all of your code. There is no gospel rule for line spacing in CSS, of course, but it's a good practice to maintain readability. 

If you choose to have no blank line between CSS selectors + properties, then be sure to stay with that rule throughout the document.